packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -39,10 +40,47 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
 
     // TODO: should be the amount sufficient to cover the gas costs of all of the transactions up
     // to and including the CALL/CREATE which forms the entrypoint of the transaction.
-    uint256 constant EXECUTION_VALIDATION_GAS_OVERHEAD = 25000;
     OVM_ETH constant ovmETH = OVM_ETH(0x4200000000000000000000000000000000000006);
 
 
+    /*********************
+     * Private Functions *
+     *********************/
+
+    /**
+     * Compute the intrinsic gas of a transaction
+     * @param _encodedTransaction Signed EIP155 transaction bytes.
+     * @param _isCreate Whether the transaction creates a contract
+     * @return The amount of intrinsic gas
+     */
+    function _intrinsicGas(
```

Comment:
Added a comment to the #882 to double check that this contract will never run out of gas (always has enough to INCREMENTNONCE). If it does this then we can close that issue!